core/net_utilities.h

```diff
@@ -256,7 +256,7 @@ struct ListenerHandle {
 	}
 
 	static ListenerHandle to_handle(const ChangesListener *p_listener) {
-		return { reinterpret_cast<std::intptr_t>(p_listener) };
+		return ListenerHandle{ reinterpret_cast<std::intptr_t>(p_listener) };
```

Comment:
Missing double {}?

Reply:
I am not sure why but this one explicitly has to be single braces.

<img width="1058" alt="Screenshot 2024-03-16 at 9 42 29 PM" src="https://github.com/GameNetworking/network_synchronizer/assets/1646875/e18c15f2-366e-48ef-93ef-c548c3d1058b">